src/components/features/Brand/Brand.js

```diff
@@ -1,42 +1,121 @@
+    }
+  }
+
+  changePageNext() {
+    let currentBrand = this.state.activePage;
+
+    if (currentBrand !== 0) {
+      this.handlePageChange(currentBrand - 1);
+    }
+  }
+
   render() {
-    const { brands } = this.props;
+    const { activePage } = this.state;
+    const { brands, rwdMode } = this.props;
+    const pagesCount = Math.ceil(brands.length / this.rwdCardsInRow[rwdMode]);
+
+    const dots = [];
+    for (let i = 0; i < pagesCount; i++) {
+      dots.push(
+        <li>
+          <a
+            onClick={() => this.handlePageChange(i)}
+            className={i === activePage && styles.active}
+          >
+            page {i}
+          </a>
+        </li>
+      );
+    }
 
     return (
-      <div className={styles.root}>
-        <div className='container'>
-          <div className={`row ${styles.marksRow}`}>
-            <div className={`col ${styles.brandBox}`}>
-              <div className={`col ${styles.thumbnail}`}>
-                <Button className={styles.arrowButton}>
-                  <i className={[styles.arrow, styles.left].join(' ')}></i>
-                </Button>
+      <Swipeable leftAction={this.leftAction} rightAction={this.rightAction}>
+        <div className={styles.root}>
+          <div className='container'>
+            <div className={styles.panelBar}>
+              <div className='row no-gutters align-items-end'>
+                <div className={'col-auto ' + styles.heading}>
+                  <h3>Brand Marks</h3>
+                </div>
+                <div className={'col-auto ' + styles.dots}>
+                  <ul>{dots}</ul>
+                </div>
               </div>
-              {brands.map(item => (
-                <div key={item.id} className={`col ${styles.mark}`}>
-                  <img className={styles.photo} alt='brand_mark' src={item.mark}></img>
+            </div>
+
+            <div className={`row ${styles.marksRow}`}>
+              <div className={`col ${styles.brandBox}`}>
+                <div className={styles.brandsButtons}>
+                  <Button className={styles.button}>
+                    <FontAwesomeIcon icon={faChevronLeft}></FontAwesomeIcon>
+                  </Button>
+                </div>
+                {brands
+                  .slice(
+                    activePage * this.rwdCardsInRow[rwdMode],
+                    (activePage + 1) * this.rwdCardsInRow[rwdMode]
+                  )
+                  .map(item => (
+                    <div key={item.id} className={`col ${styles.mark}`}>
+                      <img
+                        className={styles.photo}
+                        alt='brand_mark'
+                        src={item.mark}
+                      ></img>
+                    </div>
+                  ))}
+                <div className={styles.brandsButtons}>
+                  <Button className={styles.button}>
+                    <FontAwesomeIcon icon={faChevronRight}></FontAwesomeIcon>
```

Comment:
Nie działa klikanie w strzałki

Reply:
to jest w task 31, funkcjonalność tego komponentu